app/src/main/java/com/vero/tasky/agenda/data/local/entities/DeletedPhotoEntity.kt

```diff
@@ -0,0 +1,24 @@
+package com.vero.tasky.agenda.data.local.entities
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.ForeignKey
+import androidx.room.PrimaryKey
+
+@Entity(
+    foreignKeys = [
+        ForeignKey(
+            entity = EventEntity::class,
+            parentColumns = arrayOf("id"),
+            childColumns = arrayOf("eventId"),
+            onDelete = ForeignKey.CASCADE
+        )
+    ]
+)
+data class DeletedPhotoEntity(
```

Comment:
What's the use of this class?

You could just save the event with less photos, and that should act the same way as deleting the other photos

Reply:
If this will be triggered, I should know what images were deleted 
![image](https://github.com/AndreVero/Tasky/assets/39765531/3224b37f-20a6-435a-b0ce-911489b4cc2a)

Reply:
To update event later

Reply:
Here I use it 
![image](https://github.com/AndreVero/Tasky/assets/39765531/67a3104b-88b1-4c4f-bf4e-73541c827b59)